GSA/made-in-america
# [MIA-212] : updates piids conditional and new app build

## Description

A bug was discovered on stage that indicated the mapping of data in our submitted data was not working as expected.

This works to update the conditional so we don't point to the first index unless piids key exists.

## How to test

- navigate to https://federalist-c275c827-2779-42ff-ac19-0331ebcacca9.app.cloud.gov/preview/gsa/made-in-america/bug/gsa-fix-piids-conditional/waivers/
- select 'submitted'
- confirm that accordions are working as expected, and data is populated as expected

added cypress test 
-npm run start
-spilt terminal
-npm run test(will run all tests locally)
-or npm run cy:open (will open test runner and you can just the results-display test)

To prepare for code review, please follow the following checks.
